discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -127,7 +127,15 @@ def set_network_params(
         vip_dhcp_allocation=env_variables['vip_dhcp_allocation'],
         cluster_machine_cidr=env_variables['machine_cidr']
     ):
-        self.api_client.update_cluster(self.id, {"vip_dhcp_allocation": vip_dhcp_allocation})
+        cluster_cidr=env_variables['cluster_cidr']
+        service_cidr=env_variables['service_cidr']
+        host_prefix=env_variables['host_prefix']
+        self.api_client.update_cluster(self.id, {
+            "vip_dhcp_allocation": vip_dhcp_allocation,
+            "service_network_cidr": service_cidr,
+            "cluster_network_cidr": cluster_cidr,
+            "cluster_network_host_prefix": host_prefix,
```

Comment:
You don't have to set it into a variable. Just set the dictionary directly
```python
        self.api_client.update_cluster(self.id, {
            "vip_dhcp_allocation": vip_dhcp_allocation,
            "service_network_cidr": env_variables['service_cidr'],
            "cluster_network_cidr": env_variables['cluster_cidr'],
            "cluster_network_host_prefix": env_variables['host_prefix'],
    })
```

Reply:
OK